src/screens/DetailDomain/DetailDomainScreen.js

```diff
@@ -327,10 +326,10 @@ const styles = StyleSheet.create({
   textDesc: {
     fontFamily: fonts.inter[400],
     fontSize: 16,
-    color: COLORS.black
+    color: '#000'
   },
   more: {
-    color: COLORS.blueZaffre,
+    color: '#0e24b3',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Hardcoded color values should be replaced with constants for better maintainability and consistency across the application. If the color needs to be changed in the future, it would be easier to change it in one place (the constant) rather than searching through the entire codebase.
```diff
-    color: '#000'
+    color: colors.black,
-    color: '#0e24b3',
+    color: colors.blueZaffre,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->